game/runtime.cpp

```diff
@@ -357,9 +357,9 @@ RuntimeExitStatus exec_runtime(GameLaunchOptions game_options, int argc, const c
     try {
       Gfx::Loop([]() { return MasterExit == RuntimeExitStatus::RUNNING; });
     } catch (std::exception& e) {
-      fmt::print("Exception thrown from graphics loop: {}\n", e.what());
-      fmt::print("Everything will crash now. good luck\n");
-      throw;
+      lg::error("Exception thrown from graphics loop: {}\n", e.what());
+      lg::error("Everything will crash now. good luck\n");
+      throw e;
```

Comment:
this should be just `throw;`, not `throw e;`
(see https://stackoverflow.com/questions/1095225/exception-slicing-is-this-due-to-generated-copy-constructor)

If you do `throw e`, it will copy the exception using the copy-constructor for `std::exception` (bad, causing slicing). But `throw` will rethrow the exception without copying.